.github/workflows/rawls-run-azure-e2e-tests.yaml

```diff
@@ -16,23 +28,47 @@ on:
         required: true
         default: true
         type: boolean
+      owner-subject:
+        description: 'Owner subject (used for creating billing project in E2E testing)'
+        required: true
+        default: 'hermione.owner@quality.firecloud.org'
+        type: string
+      student-subjects:
+        description: 'A JSON array of Student subjects used for E2E testing'
+        required: true
+        default: '["harry.potter@quality.firecloud.org","ron.weasley@quality.firecloud.org"]'
+        type: string
+      service-account:
+        description: 'Email address or unique identifier of the Google Cloud service account for which to generate credentials'
+        required: true
+        default: 'firecloud-qa@broad-dsde-qa.iam.gserviceaccount.com'
```

Comment:
has this SA been tested? I copied this code from elsewhere and ran into a WIF-not-enabled issue and I don't see an entry in terraform to enable it.

Reply:
The `WIF` isn't used in `rawls-run-azure-e2e-tests.yaml`, at least for now there's not a need for it because everything is taken care of in `terra-github-workflows`. That's why I decided not to include a terraform entry for it.